### DISCUSSION TITLE
What is the default understanding of the offices?

### DISCUSSION THREAD

**Discussion by MRoth1910** [DivinumOfficium/divinum-officium#4113] [Q&A]
The title isn't so clear, so let me explain.

Matins, Prime, and Compline have "Jube, Domne," benedicere" before the readings where the lector asks for a blessing of the officiant in choir. But the breviary is clear: Extra Chorum, quando ab uno tantum recitatur Offiicum, ante singulas Lectiones Matutini atque ad Lectionem brevem Primae et Completorii, dicitur: Jube, Domine, benedicere; et subjungitur congruens Benedictio.

However, the collects are always preceded by "Domine, exaudi orationem meam" etc. as if it was a choir of nuns or of clerics not yet deacons, particularly for DA and before where even priests recite "Dominus vobiscum" when alone and outside of choir. We may not like that people including clerics say the DA office exclusively, but what they do isn't for us to police, and the site should actually reflect the rubrics as they are, regardless of the juridical status of the pre-1954 breviary. I know that there is a box to check for "priest" (but this doesn't make a ton of sense either, because in the 1960 office, the priest only says "Dominus vobiscum" in public, and IMHO, if accuracy is the goal, then the label should say "Priest (deacon (DA/Tridentine)".

So what currently happens is not entirely coherent to me if the idea that the office is as if it's being recited alone by someone not a cleric…and let's be honest, I don't think that it's obvious to change from "Domne" to "Domine", even for someone fairly skilled in Latin, with the ideal level to **read** the breviary in Latin — which is why a rubric exists.

It's currently a mish-mash of the choir rubrics, which I grant are the default texts of the breviary in the Ordinary and, at Prime and at Compline, even in the psalter, and of private recitation.

**Comment by fiapps** [comment]
I understand the `priest` option to mean that a priest is leading the office as hebdomadarian. This understanding works with the 1960 rubrics and would mean "Jube, domne" should be used when `priest != 0`. I'm not familiar with the older rubrics, so I don't know what should happen if a deacon leads the office or a priest or deacon prays the office on his own. In any case, I doubt the flag can be given a meaning that a short label will fully explain.

**Comment by MRoth1910** [comment]
> In any case, I doubt the flag can be given a meaning that a short label will fully explain.

TBH, I do not especially care, since at some point, people need to know the rubrics, **but** at the same time, there should not be a mishmash. If there is no hebdom, then you should not have "Domne", and if you are alone, and not a priest before 1962, you should certainly not have "Jube Domne" but then "Domine exaudi". The rubrics do not say what to do for the blessings in a choir of nuns using the Roman office, unlike the rubrics for Compline in the Liber Usualis which are actually lifted from the antiphonal — but that's a case where you use "Domine exaudi". I believe that deacons serving as officiant (but not celebrant; they can't bless incense in the usus antiquior, so they can't serve as celebrant of a solemn or semi-solemn office) also said "Dominus vobiscum", at least publicly. They say it elsewhere…

But the site's original workings are based on DA as the norm, sort of hacked to make 1960 the norm.

**Comment by fiapps** [comment]
For what it's worth, in the app I label the `priest` option "Dominus vobiscum," and explain it like this in the help for the app:

> With the 1960 rubrics, this should be on when a priest leads the office and prays with others. With older rubrics, a priest should turn it on even when he prays alone. When it is on, the dialogs at the end of the office begin with the priest saying “Dominus vobiscum.” Otherwise, the dialogs begin “Domine exaudi orationem meam.” Of course, if you’re sufficiently familiar with the office, you can just leave the switch alone and make the adjustments mentally.

So I actually did have an interpretation of it for DA and other old rubrics, contrary to what I said above.

The help for the website just says, "Check or uncheck the **Priest** box, so you will recite the _Dominus vobiscum_ or the _Domine exaudi_ as appropriate."

**Reply by MRoth1910** [reply]
App? It's too bad that BrevMeum is basically dead. The new calendars are not available. But the texts are right.

Also [deacons use this too](Gentlemen, prior to the reform of 1960, it was always strictly mandatory for all those who said the Office privately AND were ordained to at least the diaconate to say "Dominus vobiscum" at the appropriate places, and the reply "Et cum spiritu tuo" back to themselves, if they were saying the Office alone. This is still the case to this day in the EF Mass. Explanation here:

http://www.newliturgicalmovement.org/2017/02/st-peter-damian-on-liturgical-prayer.html).

I guess my point is, particularly at Compline where it's literally the first thing that you see, the office should be coherent. It is currently not so.

**Comment by FAJ-Munich** [comment]
At least for the website, there is no issue to introduce new options, like I have recently to show Flexa and inline verse numbers in the Psalms for those that want it. 
The issue is more on making it known to the users that there are new features. 

Basically, there are four combinations, aren't there?
- priest/deacon alone:
Jube Domine + Dominus vobiscum

- priest/deacon in Community:
Jube domne + Dominus vobiscum

- "lay" community with a superior
Jube domne + Domine exaudi
(Although the vocative is wrong for a nun/Äbtissin)

- Lay" alone:
Jube Domine + Domine exaudi

**Reply by MRoth1910** [reply]
Lay/1960 alone, but yes.

**Reply by FAJ-Munich** [reply]
In fact, looking at the Preces Dominicales ad Primam and the opening of Complete, the single Confiteor presumes there is no Hebdomadarius!
This works according to the DA1942 book, _extra chorum, si unus vel duo tantum recitent Officium, et in Choro Monialium ..._

Comparing the rubric you cited at the first instance of "Jube, domne", interestingly, the 1906 book doesn't even have one.

**Reply by MRoth1910** [reply]
Yes. The nun sings "Domine", although the "Domine exaudi" rubric is essentially made for nuns' choirs above all. I just asked Gregory DiPippo.

**Reply by FAJ-Munich** [reply]
Picking up our conversation, the final piece, which is unclear to me, is what to do for the Tridentine versions? 

- For DA, the rubric at the first Jube is clear: if you are not in choir (and for nuns) it is "Domine". 
- For Trid, this very rubric is not printed in the books. 

Obviously, nobody would have had anticipated anyone not bound to the office to pray it on his own. Outside of monasteries, the obligation began with being raised to the Subdiaconate. These would have learned the office "in choro" in the seminary, wouldn't they? But what did priests in remote villages do? Domne or Domine? The books only indicated the former.

**Reply by APMarcello3** [reply]
@FAJ-Munich Most secular clergy in the seminaries would not have had the full office in choir/common, except perhaps Sunday Vespers and perhaps occasionally Compline. Most clergy prior to the Council might have had a course on the breviary rubrics, but even that did not always occur.

**Comment by FAJ-Munich** [comment]
PR #4905 is going to add the necessary Rubric to Jube, domne for both Divino Afflatu versions as per the Moanstic 1930 and Roman 1942 PDF scans I have available.
